docs/api.md

```diff
@@ -1848,6 +1881,157 @@ This endpoint supports retrieving extended metadata; provide a header `X-Extende
 + Response 403 (application/json)
     + Attributes (Error 403)
 
+### Getting Submission metadata [GET /v1/projects/{projectId}/forms/{xmlFormId}/submissions/{instanceId}]
+
+Like how `Form`s are addressed by their XML `formId`, individual `Submission`s are addressed in the URL by their `instanceId`.
```

Comment:
It also feels like it would be helpful to call out here that the instance id is the id of the original submission and that only the latest version is returned. Maybe the section introduction could have that but it seems important enough to me to repeat in each section so they truly stand alone.

Reply:
i try to keep these sections short, because of the apiary layout. the main bar is really meant to be more of an index. a lot of content and the reading gets difficult and cramped. the parameter descriptions themselves (i think..) always describe exactly what is happening in each case. but i'll add it to the section introduction as well.